drams/local-settings.py

```diff
@@ -1,5 +1,11 @@
 from .base_settings import *
+import os
 
 MIDDLEWARE += ['whitenoise.middleware.WhiteNoiseMiddleware', ]
 
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedStaticFilesStorage'
+
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
```

Comment:
I don't think this is needed as it's in `base_settings.py` which is imported.

Reply:
All these global variables must be a Django thing!

Reply:
Removed